test_package/conanfile.py

```diff
@@ -15,8 +16,10 @@ def build(self):
         cmake.build()
 
     def test(self):
+        arm_archs = ['arm', 'aarch64_be', 'aarch64', 'armv8b', 'armv8l']
```

Comment:
better to use `tools.cross_building` instead (https://docs.conan.io/en/latest/reference/tools.html?highlight=cross_building#tools-cross-building)

Reply:
Actually it's a bit too generic: the current version prevent x86 tests to run on x64 systems so I will change that.